packages/ember-runtime/lib/system/core_object.js

```diff
@@ -930,17 +907,4 @@ CoreObject.ClassMixin = ClassMixin;
 
 ClassMixin.apply(CoreObject);
 
-CoreObject.reopen({
-  didDefineProperty(proto, key, value) {
```

Comment:
The hook not this method but until we make hooks in meta we probably want to keep an empty hook here